src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+
+import { FormType, TodoType } from '../../types';
+import usersFromServer from '../../api/users';
+import { findMaxId } from '../../helpers/findMaxTodoId';
+
+type Props = {
+  addTodo: (todo: TodoType) => void,
+  todos: TodoType[]
+};
+
+const initialFormValue = {
+  titleInput: '',
+  userSelect: 0,
+};
+
+export const TodoForm: React.FC<Props> = ({ addTodo, todos }) => {
+  const [formValues, setFormValues] = useState<FormType>(initialFormValue);
+  const [inputError, setInputError] = useState(false);
+  const [selectError, setSelectError] = useState(false);
```

Comment:
hasInputError, same with select

Reply:
I don't agree with you. My main argument is that there are 2 separate errors for two different types of inputs and therefore there should be two stakes. If I combine the errors into one, it will be work not properly